flexmeasures/ui/templates/assets/asset_properties.html

```diff
@@ -245,152 +258,1078 @@ <h3>All child assets for {{ asset.name }}
+            "soc-max": { "units": flexModelAllowedUnits.energy, "desc": "A constant upper boundary for all values in the schedule (defaults to max soc target, if provided).", "types": "One fixed value or a dynamic signal (via a sensor)." },
+            "soc-minima": { "units": flexModelAllowedUnits.energy, "desc": "Set points that form lower boundaries, e.g. to target a full car battery in the morning (defaults to NaN values).", "types": "One fixed value or a dynamic signal (via a sensor)." },
+            "soc-maxima": { "units": flexModelAllowedUnits.energy, "desc": "Set points that form upper boundaries at certain times (defaults to NaN values).", "types": "One fixed value or a dynamic signal (via a sensor)." },
+            "soc-targets": { "units": flexModelAllowedUnits.energy, "desc": "Exact set point(s) that the scheduler needs to realize (defaults to NaN values).", "types": "One fixed value or a dynamic signal (via a sensor)." },
+            "soc-gain": { "units": flexModelAllowedUnits.power, "desc": "SoC gain per time step, e.g. from a secondary energy source (defaults to zero). This field allows multiple settings, either fixed or dynamic, which add up to an aggregated gain.", "types": "An array of One fixed value or a dynamic signal (via a sensor)." },
+            "soc-usage": { "units": flexModelAllowedUnits.power, "desc": "SoC reduction per time step, e.g. from a load or heat sink (defaults to zero). This field allows multiple settings, either fixed or dynamic, which add up to an aggregated gain.", "types": "An array of One fixed value or a dynamic signal (via a sensor)." },
+            "roundtrip-efficiency": { "units": flexModelAllowedUnits.efficiency, "desc": "Below 100%, this represents roundtrip losses (of charging & discharging), usually used for batteries. Can be percent or ratio [0,1] (defaults to 100%).", "types": "Fixed value only." },
+            "charging-efficiency": { "units": flexModelAllowedUnits.efficiency, "desc": "Apply efficiency losses only at time of charging, not across roundtrip (defaults to 100%).", "types": "Fixed value only." },
+            "discharging-efficiency": { "units": flexModelAllowedUnits.efficiency, "desc": "Apply efficiency losses only at time of discharging, not across roundtrip (defaults to 100%).", "types": "Fixed value only." },
+            "storage-efficiency": {
+                "units": flexModelAllowedUnits.efficiency,
+                "desc": "This can encode losses over time, so each time step the energy is held longer leads to higher losses (defaults to 100%). The storage efficiency (e.g. 95% or 0.95) to use for the schedule is applied over each time step equal to the sensor resolution. For example, a storage efficiency of 95 percent per (absolute) day, for scheduling a 1-hour resolution sensor, should be passed as a storage efficiency of 0.95<sup>1/24</sup> = 0.997865",
+                "types": "Fixed value only."
```

Comment:
I believe this is wrong, as I can also choose a sensor.
```suggestion
                "types": "One fixed value or a dynamic signal (via a sensor)."
```

Reply:
This is an oversight from me, i discussed about this field with @Flix6x and he said we keep it as a fixed value, but after testing with `roundtrip-effeciency` i forgot to set the other effeciency to Fixed value only.

But we can all quickly agree on this if we want to accept both or one.

Reply:
>I set a sensor with kW unit for storage-efficiency. When the model is saved, I see:

I think this will also depend on the decison made in the comment